modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -1,14 +1,36 @@
 #include "opencv_coins.hpp"
-
+#include <string>
+#include <iostream>
+int dist_transform;
 using namespace cv;
+Mat img = imread("C:\\Users\\k_kab\\Picture\\coins.jpg");
```

Comment:
Unused line?

Reply:
It is way where can find pitcheres with coins local

Reply:
But why you need this? `countCoins` receives `img` which is the same

Reply:
ok. i find this file near main.cpp. I changed